seatunnel-connectors-v2/connector-paimon/src/main/java/org/apache/seatunnel/connectors/seatunnel/paimon/catalog/PaimonCatalogLoader.java

```diff
@@ -51,24 +50,35 @@ public class PaimonCatalogLoader implements Serializable {
     private static final String HDFS_IMPL_KEY = "fs.hdfs.impl";
 
     private String warehouse;
+    private PaimonCatalogEnum catalogType;
+    private String catalogUri;
 
     private PaimonHadoopConfiguration paimonHadoopConfiguration;
 
-    public PaimonCatalogLoader(PaimonSinkConfig paimonSinkConfig) {
-        this.warehouse = paimonSinkConfig.getWarehouse();
-        this.paimonHadoopConfiguration = PaimonSecurityContext.loadHadoopConfig(paimonSinkConfig);
+    public PaimonCatalogLoader(PaimonConfig paimonConfig) {
+        this.warehouse = paimonConfig.getWarehouse();
+        this.catalogType = paimonConfig.getCatalogType();
+        this.catalogUri = paimonConfig.getCatalogUri();
+        this.paimonHadoopConfiguration = PaimonSecurityContext.loadHadoopConfig(paimonConfig);
     }
 
     public Catalog loadCatalog() {
         // When using the seatunel engine, set the current class loader to prevent loading failures
         Thread.currentThread().setContextClassLoader(PaimonCatalogLoader.class.getClassLoader());
         final Map<String, String> optionsMap = new HashMap<>(1);
-        optionsMap.put(WAREHOUSE.key(), warehouse);
-        final Options options = Options.fromMap(optionsMap);
+        optionsMap.put(CatalogOptions.WAREHOUSE.key(), warehouse);
         if (warehouse.startsWith(HDFS_PREFIX)) {
             checkConfiguration(paimonHadoopConfiguration, HDFS_DEF_FS_NAME);
             paimonHadoopConfiguration.set(HDFS_IMPL_KEY, HDFS_IMPL);
         }
+        if (PaimonCatalogEnum.HIVE.getType().equals(catalogType.getType())) {
```

Comment:
```suggestion
        if (PaimonCatalogEnum.HIVE.getType().equalsIgnoreCase(catalogType.getType())) {
```

Reply:
> 

There is no need to change this, because both are enumerations, and the type obtained is the same.

Reply:
ok